pages/account/home.js

```diff
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types'
-import React, { useMemo } from 'react'
+import React, { useMemo, useState,useEffect } from 'react'
```

Comment:
```suggestion
import React, { useMemo, useState, useEffect } from 'react'
```
Needs to have a space between as it is causing the build to fail